### PR TITLE
Add ui style for "Open a blank issue" button

### PR DESCRIPTION
**Pull request by a1012112796**
From:
tmp1

To:
image

### CODE REVIEW

**Comment by silverwind** [comment]
Certainly better than before but I think even better would be to move the button to the right, e.g. `Blank Issue <whitespace> Get Started`.

**Comment by jolheiser** [comment]
I believe the reason GH does it as a link is because users should be drawn to and encouraged to use the templates if present, rather than making a default issue.

But I am admittedly not a UI pro. 😅

File: templates/repo/issue/choose.tmpl [hunk 1]
```diff
@@ -7,23 +7,31 @@
...
+				<div class="column right aligned">
+					<a href="{{.RepoLink}}/issues/new{{if .milestone}}?milestone={{.milestone}}{{end}}"
+						class="ui green button">{{$.i18n.Tr "repo.issues.choose.get_started"}}</a>
+				</div>
 			</div>
 		</div>
 	</div>
 </div>
-{{template "base/footer" .}}
+{{template "base/footer" .}}
```
Comment (zeripath) [inline templates/repo/issue/choose.tmpl]:
```suggestion
{{template "base/footer" .}}

```

Comment (a1012112796) [reply templates/repo/issue/choose.tmpl]: May be Stale.

**Comment by silverwind** [comment]
> encouraged to use the templates

We could maybe de-emphasize the button by making it gray (e.g. remove the `green` class).

**Review by silverwind — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -940,7 +940,8 @@ issues.new.no_assignees = No Assignees
 issues.new.no_reviewers = No reviewers
 issues.new.add_reviewer_title = Request review
 issues.choose.get_started = Get Started
-issues.choose.blank = Open a blank issue
+issues.choose.blank = Blank issue
+issues.choose.blank_about = Create an issue without a template.
```
Comment (jolheiser) [inline options/locale/locale_en-US.ini]: There may or may not still be an issue template at play, as default will still look for the legacy template.

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
:rocket:

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
this looks so mouch better

**Comment by 6543** [comment]
ready for 🚀

**Comment by lafriks** [comment]
🚀

**Pull request merged**